test/hca-rjsf/config/medicare.unit.spec.jsx

```diff
@@ -0,0 +1,111 @@
+  const { schema, uiSchema } = formConfig.chapters.insuranceInformation.pages.medicare;
+  it('should render', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          definitions={formConfig.defaultDefinitions}
+          uiSchema={uiSchema}/>
+    );
+    const formDOM = findDOMNode(form);
+
+    expect(formDOM.querySelectorAll('input').length)
+      .to.equal(4);
+  });
+
+  it('should not submit empty form', () => {
+    const onSubmit = sinon.spy();
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          definitions={formConfig.defaultDefinitions}
+          onSubmit={onSubmit}
+          uiSchema={uiSchema}/>
+    );
+
+    const formDOM = findDOMNode(form);
+
+    submitForm(form);
+
+    expect(formDOM.querySelectorAll('.usa-input-error').length).to.equal(2);
+    expect(onSubmit.called).to.be.false;
+  });
+
+  it('should reveal required effective date', () => {
+    const onSubmit = sinon.spy();
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          definitions={formConfig.defaultDefinitions}
+          onSubmit={onSubmit}
+          uiSchema={uiSchema}/>
+    );
+
+    const formDOM = findDOMNode(form);
+
+    expect(formDOM.querySelectorAll('input, select').length).to.equal(4);
+
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_isEnrolledMedicarePartAYes'), {
+      target: {
+        value: 'Y'
+      }
+    });
+
+    expect(formDOM.querySelectorAll('input, select').length).to.equal(7);
+
+    submitForm(form);
+
+    expect(formDOM.querySelectorAll('.usa-input-error').length).to.equal(2);
+    expect(onSubmit.called).to.be.false;
+  });
+  it('should require past effective date', () => {
+    const onSubmit = sinon.spy();
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          definitions={formConfig.defaultDefinitions}
+          onSubmit={onSubmit}
+          uiSchema={uiSchema}/>
+    );
+
+    const formDOM = findDOMNode(form);
+
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_isEnrolledMedicarePartAYes'), {
+      target: {
+        value: 'Y'
+      }
+    });
+
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_medicarePartAEffectiveDateMonth'), {
+      target: {
+        value: '1'
+      }
+    });
+
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_medicarePartAEffectiveDateDay'), {
+      target: {
+        value: '1'
+      }
+    });
+
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_medicarePartAEffectiveDateYear'), {
+      target: {
+        value: ((new Date()).getFullYear() + 1).toString()
+      }
+    });
+
+    submitForm(form);
+
+    expect(formDOM.querySelectorAll('.usa-input-error').length).to.equal(2);
+    expect(onSubmit.called).to.be.false;
+  });
+});
```

Comment:
Should probably add one last test with required inputs filled and test that submission is successful.